flow-defs/src/bottom-sheet.js.flow

```diff
@@ -2,25 +2,29 @@
 
 import { SheetResponse } from "./post-message";
 declare type SheetListType = "SINGLE_SELECTION";
-declare type SheetListItem = {
+export type SheetListItem = $ReadOnly<{
   id: string,
   title?: string,
   description?: string,
-};
-declare type SheetUIElement = {
+  icon?: $ReadOnly<{
+    url: string,
+    urlDark?: string,
+  }>,
```

Comment:
What about prominent icons ?

Reply:
webapp can send a prominent color when needed.
BTW, isn't this only for vivo?
Also prominent in moves will die in Q4 afaik

Reply:
Its not only for vivo, this is global component (plan selector). but as said by Edu G, prominent won't survive the digital hub project, so I'll approve the PR.